standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnStore.java

```diff
@@ -305,6 +305,15 @@ HeartbeatTxnRangeResponse heartbeatTxnRange(HeartbeatTxnRangeRequest rqst)
   @RetrySemantics.ReadOnly
   ShowCompactResponse showCompact(ShowCompactRequest rqst) throws MetaException;
 
+  /**
+   * Get one latest record of completed compaction by given table/partition.
```

Comment:
I think we should highlight that `READY_FOR_CLEANING` compactions are returned as well

Reply:
Added to the description.